src/atsamd/Kconfig

```diff
@@ -71,10 +80,16 @@ config MACH_SAME54
     select MACH_SAMX5
 config HAVE_SAMD_CANBUS
     bool
-    default y if MACH_SAME51 || MACH_SAME54
+    default y if MACH_SAMC21 || MACH_SAME51 || MACH_SAME54
+config HAVE_SAMD_USB
+    bool
+    default n if MACH_SAMC21G18
+    default y
+
```

Comment:
Spurious blank line added.

Reply:
Removed, not sure why this is still green...